tests/models/openconfig/data/openconfig_system/translate/ios/test_case_1/data.json

```diff
@@ -0,0 +1,98 @@
+{
+    "openconfig-system:system": {
+      "config": {
+        "hostname": "csr1000v-1"
+      },
+      "clock": {
+        "config": {
+          "timezone-name": "CST -6 0"
+        }
+      },
+      "dns": {
+        "servers": {
```

Comment:
Can we have it translate a single DNS server and multiple DNS servers? Just to make sure it works correctly.

Reply:
I added a test case for this, and yes, it was broken.  :)